pandas/tseries/offsets.py

```diff
@@ -322,37 +322,42 @@ def _params(self):
 
     def __repr__(self):
         className = getattr(self, '_outputName', type(self).__name__)
+
+        if abs(self.n) != 1:
+            plural = 's'
+        else:
+            plural = ''
+
```

Comment:
this whole thing should just be moved to a separate function, much more clear this way

Reply:
this formatting function (e.g. repr should just call it, passing parameters in).